React/Fabric/Mounting/ComponentViews/ActivityIndicator/RCTActivityIndicatorViewComponentView.h

```diff
@@ -12,7 +12,7 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /**
- * RCTUIView class for root <ActivityIndicator> component.
+ * UIView class for root <ActivityIndicator> component.
```

Comment:
I guess even comments should have tags 😅

Reply:
I reverted all the comment changes to keep the PR's cleaner.

Reply:
I would vote to keep them. If they represent diffs, I think they should represent _all_ diffs as much as we can, and a change to that guidance should be part of #1392 . Does that sound alright?

For cleanliness, we could also put opening and closing tags on line 14 and 16 but I'm not sure that's better

Reply:
I see, so we could tag the entire comment block? I'm ok with that. 

Currently I've reverted the comment changes since it was adding noise to the PR but I can add those back w/ tags if you would like.